routing_common/transit_types.hpp

```diff
@@ -218,6 +216,28 @@ class Gate
   m2::PointD m_point;
 };
 
+class ShapeId
+{
+public:
+  ShapeId() = default;
+  ShapeId(StopId id1, StopId id2) : m_stop1Id(id1), m_stop2Id(id2) {}
```

Comment:
When I proposed you to rename "stop_id1" to "id1" I also meant to rename "m_stop1Id" to "m_id1".

Reply:
And "m_stop2Id" to "m_id2".

Reply:
If you wish to keep "m_stop1Id" as is, please, revert  "id1" to "stop1Id".

Reply:
I propose to keep |m_stop1Id| and |m_stop2Id|. Let's revert the param names.